babel.config.js

```diff
@@ -24,6 +24,10 @@ module.exports = {
         ],
         'babel-plugin-px-to-rem',
         '@babel/plugin-transform-runtime',
+        ["prismjs", {
+            // This list should match those defined in typescript type Language
+            "languages": ["typescript", "javascript", "css", "markup", "scala", "elm"]
```

Comment:
For the record, here's the list currently supported by composer.

![image](https://user-images.githubusercontent.com/8754692/104306860-68984700-54c6-11eb-8b97-9aa5f8d5e980.png)

Reply:
We should probably add `typescript` to that list. Is that easy to do?

Reply:
Not sure, first implementation in composer is here: https://github.com/guardian/flexible-content/pull/459. That makes it look like it's using [pretiffy](https://github.com/googlearchive/code-prettify) which may or may not support TS. To be fair, it might not matter that much.

Reply:
Prettify is [deprecated](https://github.com/googlearchive/code-prettify) and I'm not sure if the implementation we have with it on Frontend is working anymore - the code blocks I see there don't seem to have much styling other than a basic pass.